modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/server/gwt/BundleGWTServiceImpl.java

```diff
@@ -71,7 +71,7 @@ public ResourceTypeBundleConfiguration getResourceTypeBundleConfiguration(int co
     public BundleVersion createOrStoreBundleVersionViaURL(String url, String username, String password)
         throws RuntimeException {
         try {
-            BundleVersion results = bundleManager.createBundleVersionViaURL(getSessionSubject(), url, username,
+            BundleVersion results = bundleManager.createOrStoreBundleVersionViaURL(getSessionSubject(), url, username,
```

Comment:
Unless I misunderstood your comment - when you say you "modified  createBundleVersionViaURL (changed to "createOrStoreBundleVersionViaURL") " I assume that meant you renamed the API. If you just changed what method you called and that is required, that's OK. I just thought you changed the public API name - in which case, not OK.